static/katex.less

```diff
@@ -574,20 +554,9 @@
     .mtable {
         .vertical-separator {
             display: inline-block;
-            margin: 0 -0.025em;
-            width: 0.05em;
-
-            // Set a min-width so it will display at least one screen pixel.
-            @media screen {
-                min-width: 1px;
-            }
-
-            // Modify min-width for retina displays. Don't force two screen pixels.
-            @media screen and (-webkit-min-device-pixel-ratio: 2.0),
-                screen and (min-resolution: 192dpi) {
-                min-width: 0.5px;
-            }
-        }
+            margin: 0 -0.125em;     // The ink in the rule is actually 0.05em wide.
+            width: 0.25em;          // We put it in a 0.25em wide span to overcome
+        }				            // a Chrome rendering issue.
```

Comment:
Were the vertical separators disappearing at times?  Won't this change to the styling affect the horizontal layout slightly?

Reply:
Issue #970 reported a disappearing vertical separator line, so this is a valid action to take.

Re: horizontal layout: Good catch. I changed the negative `margin-left` amount, so the line has the correct space to its left. But the wider span must widen the space to its right.  No instant, obvious fix comes to mind. I'll think it over.